lib/spack/env/fj/fcc

```diff
@@ -0,0 +1 @@
+../cc
```

Comment:
Having both `FCC` and `fcc` in the same directory is a recipe for disaster on macOS where the filesystem is case-insensitive by default. Can you move `FCC` to a subdirectory called `case-insensitive`? The `cce` compiler does the same thing.

Reply:
Thanks for your reply.
As I received your indication, I changed location of FCC.
Please check commit of "Chenges location of FCC to subdirectory called case-insensitive".